fix(TBD-9317): Remove Pig components

sponomarova opened the pull request:
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)



**What is the new behavior?**



**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

amedvedenko commented:
@sponomarova I just checked out your branch and see a lot of directories containing pig in the name. Could you please take care of them or confirm that they shouldn't be removed? P.S. bash command I used: `find . -type d | grep pig | uniq`

2. ./tcommon-studio-se/main/plugins/org.talend.librariesmanager/templates/pigudf
3. ./tbd-studio-se/main/plugins/org.talend.designer.pigmap
4. ./tbd-studio-se/main/plugins/org.talend.repository.pigudf
5. ./tbd-studio-se/main/plugins/org.talend.libraries.pig
6. ./tbd-studio-se/test/plugins/org.talend.designer.pigmap.test
7. ./tbd-studio-se/i18n/plugins/org.talend.repository.pigudf.nl
8. ./tbd-studio-se/i18n/plugins/org.talend.designer.pigmap.nl
9. ./tbd-studio-ee/main/tools/pom_generator/template/hive-hcatalog-pig-adapter
10. ./tbd-studio-ee/main/tools/pom_generator/template/pig
11. ./tbd-studio-ee/main/tools/pom_generator/template/parquet-pig-bundle
12. ./tbd-studio-ee/main/tools/pom_generator/template/piggybank

amedvedenko commented:
There are also a lot of matches by `pug` keyword in the source code (like in the list below) but if it's possible - I'll review them after fixing of/answering to comment above.  

tbd-studio-ee/main/components_libs/talend-bigdata-launcher-lib/platform/src/main/java/org/talend/bigdata/launcher/utils/JobType.java
tbd-studio-ee/main/components_libs/talend-bigdata-launcher-lib/qubole/src/main/java/org/talend/bigdata/launcher/qubole/QubolePigClient.java

amedvedenko commented:
Could you please look at the files listed below? (find . | grep pig)
./tbd-studio-se/main/plugins/org.talend.libraries.hadoop.mapr.5.0.0.jars/lib/hive-hcatalog-pig-adapter-1.0.0-mapr.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hadoop.mapr.5.0.0.jars/lib/pig-core-h2.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hadoop.mapr.5.0.0.jars/lib/piggybank-mapr50X.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hadoop/lib/parquet-pig-bundle-1.6.0.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hadoop/lib/parquet-pig-bundle-1.3.2.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hadoop/lib/parquet-pig-bundle-1.5.0-cdh5.4.0.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hcatalog/lib/hive-hcatalog-pig-adapter-1.2.1.2.3.2.0-2950.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hcatalog/lib/hive-hcatalog-pig-adapter-0.14.0.2.2.0.0-2041.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hcatalog/lib/hive-hcatalog-pig-adapter-0.12.0-cdh5.1.2.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hcatalog/lib/hive-hcatalog-pig-adapter-1.1.0-cdh5.4.0.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hcatalog/lib/hive-hcatalog-pig-adapter-0.13.0-mapr-1501.jar
./tbd-studio-se/main/plugins/org.talend.libraries.hcatalog/lib/hive-hcatalog-pig-adapter-0.13.0-mapr-1408.jar

aparent-talend commented:
We need a build to confirm it's not breaking anything. Then we'll be able to do a proper review :)

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5100/plugin.xml:
```diff
@@ -428,51 +413,45 @@
 
         <libraryNeeded
-            context="plugin:org.talend.libraries.pig"
+            context="plugin:org.talend.hadoop.distribution.cdh5100"
```
why do you keep those libraries, shouldn't they be removed ?

sponomarova replied:
Didn't quite get a question. 
org.talend.libraries.pig was removed.

AlixMetivier replied:
library was needed for libraries.pig, but instead of removing it you changed the context, why not just removing ?

sponomarova replied:
Agree. Will correct that.

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5100/src/main/java/org/talend/hadoop/distribution/cdh5100/CDH5100Distribution.java:
```diff
@@ -324,22 +315,7 @@ public boolean doSupportEmbeddedMode() {
     }
-
-    @Override
-    public boolean doSupportHCatalog() {
```
why is this removed ?

sponomarova replied:
https://github.com/Talend/tbd-studio-se/blob/f63f88c0a0cc9278c64480a2a5b1dbc22ea4866c/main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/component/PigComponent.java#L24
was removed as a part of PigComponent which was deleted.

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5100/src/main/java/org/talend/hadoop/distribution/cdh5100/CDH5100Distribution.java:
```diff
@@ -324,22 +315,7 @@ public boolean doSupportEmbeddedMode() {
-    }
-
-    @Override
-    public boolean doSupportHBase() {
```
same here ?

sponomarova replied:
the same - PigComponent clean up

AlixMetivier replied:
yes but we still have tHbaseConfiguration for example so why do you remove it ?

sponomarova replied:
Sorry Initially I didn't get your point :(
you are right. I will move 
`public boolean doSupportHBase() ` to HBaseComponent interface and override this method in distributions.

sponomarova replied:
@AlixMetivier I tried to dive deeper and it seems here
`public boolean doSupportHBase()` is **true** 
only when data is loaded from **HBase** and stored to **HBase** using pig scripts.
In other cases this method is not used. 

We have **HBaseComponent** inferface to handle **tHbaseConfiguration** purposes.

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh510mr1/src/main/java/org/talend/hadoop/distribution/cdh510mr1/CDH510MR1Distribution.java:
```diff
@@ -114,21 +113,7 @@ public boolean doJavaAPISqoopImportAllTablesSupportExcludeTable() {
     }
 
-    @Override
-    public boolean doSupportHCatalog() {
```
i'll name this : Q1=why is this removed ?

sponomarova replied:
the same - PigComponent clean up

sponomarova replied:
Will move
`public boolean doSupportHCatalog() {` 
to HCatalogComponent interface.

sponomarova replied:
@AlixMetivier , the same with 
`public boolean doSupportHCatalog()`
It is only used for working with HCatalog via pig scripts. 
Since Pig is removed I don't see a purpose to keep this functionality.

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh510mr1/src/main/java/org/talend/hadoop/distribution/cdh510mr1/CDH510MR1Distribution.java:
```diff
@@ -114,21 +113,7 @@ public boolean doJavaAPISqoopImportAllTablesSupportExcludeTable() {
-    }
-
-    @Override
-    public boolean doSupportHBase() {
```
Q1

sponomarova replied:
Removed as a part of PigComponent interface:
https://github.com/Talend/tbd-studio-se/blob/f63f88c0a0cc9278c64480a2a5b1dbc22ea4866c/main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/component/PigComponent.java#L29

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh510mr2/src/main/java/org/talend/hadoop/distribution/cdh510mr2/CDH510MR2Distribution.java:
```diff
@@ -137,21 +136,6 @@ public boolean doJavaAPISqoopImportAllTablesSupportExcludeTable() {
     }
 
-    @Override
-    public boolean doSupportHCatalog() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh510mr2/src/main/java/org/talend/hadoop/distribution/cdh510mr2/CDH510MR2Distribution.java:
```diff
@@ -137,21 +136,6 @@ public boolean doJavaAPISqoopImportAllTablesSupportExcludeTable() {
-    }
-
-    @Override
-    public boolean pigVersionPriorTo_0_12() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5120/plugin.xml:
```diff
@@ -434,51 +370,45 @@
 
         <libraryNeeded
-            context="plugin:org.talend.libraries.pig"
+            context="plugin:org.talend.hadoop.distribution.cdh5120"
```
Q2 is about why keeping those :p

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5120/plugin.xml:
```diff
@@ -434,51 +370,45 @@
             name="pigutil.jar"
-            mvn_uri="mvn:org.talend.libraries/pigutil/6.0.0"
-            uripath="platform:/plugin/org.talend.libraries.pig/lib/pigutil.jar">
+            mvn_uri="mvn:org.talend.libraries/pigutil/6.0.0">            
```
end of Q2

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5120/src/main/java/org/talend/hadoop/distribution/cdh5120/CDH5120Distribution.java:
```diff
@@ -325,21 +316,6 @@ public boolean doSupportStandaloneMode() {
     }
 
-    @Override
-    public boolean doSupportHCatalog() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5120/src/main/java/org/talend/hadoop/distribution/cdh5120/CDH5120Distribution.java:
```diff
@@ -325,21 +316,6 @@ public boolean doSupportStandaloneMode() {
-    }
-
-    @Override
-    public boolean doSupportHBase() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh540/.project:
```diff
@@ -1,34 +1,34 @@
-</projectDescription>
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.talend.hadoop.distribution.cdh540</name>
```
Q2

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh550/plugin.xml:
```diff
@@ -364,51 +348,45 @@
         
         <libraryNeeded
-            context="plugin:org.talend.libraries.pig"
+            context="plugin:org.talend.hadoop.distribution.cdh550"
```
beg Q2

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh550/plugin.xml:
```diff
@@ -364,51 +348,45 @@
             name="pigutil.jar"
-            mvn_uri="mvn:org.talend.libraries/pigutil/6.0.0"
-            uripath="platform:/plugin/org.talend.libraries.pig/lib/pigutil.jar">
+            mvn_uri="mvn:org.talend.libraries/pigutil/6.0.0">            
```
end Q2

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh550/src/main/java/org/talend/hadoop/distribution/cdh550/CDH550Distribution.java:
```diff
@@ -255,21 +246,6 @@ public boolean doSupportNewHBaseAPI() {
     }
 
-    @Override
-    public boolean doSupportHCatalog() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh550/src/main/java/org/talend/hadoop/distribution/cdh550/CDH550Distribution.java:
```diff
@@ -255,21 +246,6 @@ public boolean doSupportNewHBaseAPI() {
-    }
-
-    @Override
-    public boolean doSupportHBase() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh570/plugin.xml:
```diff
@@ -379,51 +363,45 @@
         
         <libraryNeeded
-            context="plugin:org.talend.libraries.pig"
+            context="plugin:org.talend.hadoop.distribution.cdh570"
```
beg Q2

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh570/plugin.xml:
```diff
@@ -379,51 +363,45 @@
             name="pigutil.jar"
-            mvn_uri="mvn:org.talend.libraries/pigutil/6.0.0"
-            uripath="platform:/plugin/org.talend.libraries.pig/lib/pigutil.jar">
+            mvn_uri="mvn:org.talend.libraries/pigutil/6.0.0">            
```
end Q2

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh570/src/main/java/org/talend/hadoop/distribution/cdh570/CDH570Distribution.java:
```diff
@@ -259,21 +250,6 @@ public boolean doSupportNewHBaseAPI() {
     }
 
-    @Override
-    public boolean doSupportHCatalog() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh570/src/main/java/org/talend/hadoop/distribution/cdh570/CDH570Distribution.java:
```diff
@@ -259,21 +250,6 @@ public boolean doSupportNewHBaseAPI() {
-    }
-
-    @Override
-    public boolean doSupportHBase() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh580/plugin.xml:
```diff
@@ -371,51 +356,45 @@
         
         <libraryNeeded
-            context="plugin:org.talend.libraries.pig"
+            context="plugin:org.talend.hadoop.distribution.cdh580"
```
beg Q2

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh580/plugin.xml:
```diff
@@ -371,51 +356,45 @@
             name="pigutil.jar"
-            mvn_uri="mvn:org.talend.libraries/pigutil/6.0.0"
-            uripath="platform:/plugin/org.talend.libraries.pig/lib/pigutil.jar">
+            mvn_uri="mvn:org.talend.libraries/pigutil/6.0.0">            
```
end Q2

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh580/src/main/java/org/talend/hadoop/distribution/cdh580/CDH580Distribution.java:
```diff
@@ -278,21 +269,7 @@ public boolean doSupportStandaloneMode() {
     }
 
-    @Override
-    public boolean doSupportHCatalog() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh580/src/main/java/org/talend/hadoop/distribution/cdh580/CDH580Distribution.java:
```diff
@@ -278,21 +269,7 @@ public boolean doSupportStandaloneMode() {
-    }
-
-    @Override
-    public boolean doSupportHBase() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh580spark2/plugin.xml:
```diff
@@ -399,51 +399,45 @@
         
         <libraryNeeded
-            context="plugin:org.talend.libraries.pig"
+            context="plugin:org.talend.hadoop.distribution.cdh580spark2"
```
beg Q2

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh580spark2/plugin.xml:
```diff
@@ -399,51 +399,45 @@
             name="pigutil.jar"
-            mvn_uri="mvn:org.talend.libraries/pigutil/6.0.0"
-            uripath="platform:/plugin/org.talend.libraries.pig/lib/pigutil.jar">
+            mvn_uri="mvn:org.talend.libraries/pigutil/6.0.0">            
```
end Q2

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh5x/CDH5xDistributionTemplate.java:
```diff
@@ -84,22 +83,7 @@ public boolean doSupportEmbeddedMode() {
     }
-
-    @Override
-    public boolean doSupportHCatalog() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh5x/CDH5xDistributionTemplate.java:
```diff
@@ -84,22 +83,7 @@ public boolean doSupportEmbeddedMode() {
-    }
-
-    @Override
-    public boolean doSupportHBase() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java:
```diff
@@ -152,21 +151,6 @@ public boolean doSupportStandaloneMode() {
     }
 
-    @Override
-    public boolean doSupportHCatalog() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.cdh5x/src/org/talend/hadoop/distribution/cdh6x/CDH6xDistributionTemplate.java:
```diff
@@ -152,21 +151,6 @@ public boolean doSupportStandaloneMode() {
-    }
-
-    @Override
-    public boolean doSupportHBase() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.custom/src/main/java/org/talend/hadoop/distribution/custom/CustomDistribution.java:
```diff
@@ -117,26 +116,6 @@ public boolean doJavaAPISqoopImportAllTablesSupportExcludeTable() {
     }
 
-    @Override
-    public boolean doSupportHCatalog() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.custom/src/main/java/org/talend/hadoop/distribution/custom/CustomDistribution.java:
```diff
@@ -117,26 +116,6 @@ public boolean doJavaAPISqoopImportAllTablesSupportExcludeTable() {
-    }
-
-    @Override
-    public boolean doSupportHBase() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.emr450/src/main/java/org/talend/hadoop/distribution/emr460/EMR460Distribution.java:
```diff
@@ -111,11 +96,7 @@ public boolean doSupportNewHBaseAPI() {
     }
 
-    @Override
-    public boolean doSupportHBase() {
```
Q1

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.emr450/src/main/java/org/talend/hadoop/distribution/emr460/EMR460Distribution.java:
```diff
@@ -79,19 +70,13 @@ public EMR460Distribution() {
         Map<NodeComponentTypeBean, Set<DistributionModuleGroup>> result = super.buildNodeModuleGroups(distribution, version);
-
-        // Override few module groups to enable HBase features
-        result.put(new NodeComponentTypeBean(ComponentType.PIG, PigOutputConstant.PIGSTORE_COMPONENT),
```
doesn't this mean that there will be missing dependencies for HBase since we used Pig's one ?

sponomarova replied:
It is only about HBase + Pig dependencies.

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.emr5150/src/main/java/org/talend/hadoop/distribution/emr5150/EMR5150Distribution.java:
```diff
@@ -319,25 +303,11 @@ public String generateSparkJarsPaths(List<String> commandLineJarsPaths) {
     }
 
-	@Override
-	public boolean doSupportHCatalog() {
```
Q1

sponomarova replied:
Removed as a part of PigComponent:
https://github.com/Talend/tbd-studio-se/blob/f63f88c0a0cc9278c64480a2a5b1dbc22ea4866c/main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/component/PigComponent.java#L24

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.emr5150/src/main/java/org/talend/hadoop/distribution/emr5150/EMR5150Distribution.java:
```diff
@@ -319,25 +303,11 @@ public String generateSparkJarsPaths(List<String> commandLineJarsPaths) {
 	}
 
-	@Override
-	public boolean doSupportHBase() {
```
Q1

sponomarova replied:
Removed as a part of PigComponent interface:
https://github.com/Talend/tbd-studio-se/blob/f63f88c0a0cc9278c64480a2a5b1dbc22ea4866c/main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/component/PigComponent.java#L29

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.emr550/src/main/java/org/talend/hadoop/distribution/emr550/EMR550Distribution.java:
```diff
@@ -289,26 +276,6 @@ public String generateSparkJarsPaths(List<String> commandLineJarsPaths) {
-    }
-
-    @Override
-    public boolean doSupportHBase() {
```
Q1

sponomarova replied:
Removed as a part of PigComponent interface:
https://github.com/Talend/tbd-studio-se/blob/f63f88c0a0cc9278c64480a2a5b1dbc22ea4866c/main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/component/PigComponent.java#L29

AlixMetivier commented on main/plugins/org.talend.hadoop.distribution.emr550/src/main/java/org/talend/hadoop/distribution/emr550/EMR550Distribution.java:
```diff
@@ -289,26 +276,6 @@ public String generateSparkJarsPaths(List<String> commandLineJarsPaths) {
     }
 
-    @Override
-    public boolean doSupportHCatalog() {
```
Q1

sponomarova replied:
Removed as a part of PigComponent:
https://github.com/Talend/tbd-studio-se/blob/f63f88c0a0cc9278c64480a2a5b1dbc22ea4866c/main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/component/PigComponent.java#L24

sponomarova commented:
@amedvedenko , @AlixMetivier I merged with master, resolved merge conflicts, and addressed some comments/responded.
Could you guys please take a look once again when time permits?  Thanks

aparent-talend commented:
this commit needs to be reverted please : https://github.com/Talend/tbd-studio-se/pull/1125/commits/b20162b3f08e2ae37dfa64dc1d74db912b4221e4

amedvedenko commented:
main/plugins/org.talend.designer.components.bigdata/components/tPigLoad/* should be removed as well

amedvedenko commented on main/plugins/org.talend.hadoop.distribution.dataproc11.jars/.project:
```diff
@@ -1,17 +1,17 @@
-	<name>tbd-target-platform-configuration</name>
+	<name>org.talend.hadoop.distribution.dataproc11.jars</name>
 	<comment></comment>
 	<projects>
```
btw, does this is your local project?)
Also, could you please confirm that you don't touch/delete .project in .dataproc11.jars/.project file?

amedvedenko commented on tbd-target-platform-configuration/pom.xml:
```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
could you please confirm it's your local project?

sponomarova replied:
Delete will be reverted.

amedvedenko commented on tbd-target-platform-configuration/tbd-target-platform-configuration.target:
```diff
@@ -1,211 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
could you please confirm it's your local project?

sponomarova replied:
Delete will be reverted.

amedvedenko commented on podTemplate.yaml:
```diff
@@ -1,16 +0,0 @@
-apiVersion: v1
-kind: Pod
-spec:
-  containers:
```
shouldn't be removed as well

sponomarova replied:
Delete will be reverted.

amedvedenko commented on Jenkinsfile:
```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/groovy
-
-/**
- * Build pipeline for the remote-engine project.
```
shouldn't be deleted

sponomarova replied:
Delete will be reverted.

sponomarova commented on main/plugins/org.talend.hadoop.distribution.cdh4mr2/.project:
```diff
@@ -1,34 +1,28 @@
-</projectDescription>
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.talend.hadoop.distribution.cdh4mr2</name>
```
We won't be having "org.talend.libraries.pig". That's why it should be moved.

sponomarova commented on main/plugins/org.talend.hadoop.distribution.apache100/src/main/java/org/talend/hadoop/distribution/apache100/Apache100Distribution.java:
```diff
@@ -117,22 +107,7 @@ public boolean doSupportCrossPlatformSubmission() {
-    public boolean doSupportHBase() {
-        return true;
-    }
-
```
@amedvedenko , could you pls clarify why this shouldn't be removed? 
It is only about working with HBase through pig scripts. 
Is was a part of PigComponent interface which is about to be removed.

sponomarova replied:
This API is a part of PigComponent and is used only to be able to run HBase through pig scripts.
Since pig is not supported. `public boolean doSupportHBase()` is not in use any more.

sponomarova commented on main/plugins/org.talend.hadoop.distribution.cdh4mr1/.project:
```diff
@@ -1,34 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.talend.hadoop.distribution.cdh4mr1</name>
+	<comment></comment>
```
"org.talend.libraries.pig" won't be available any longer.
Actually it will be removed in scope of decreasing Studio build size.

sponomarova replied:
"org.talend.libraries.pig" is about to be removed from repository in order to reduce Studio build size.

sponomarova commented:
> * components/tPigLoad

Will be removed.

sponomarova commented on main/plugins/org.talend.hadoop.distribution.emr550/src/main/java/org/talend/hadoop/distribution/emr550/EMR550Distribution.java:
```diff
@@ -289,26 +276,6 @@ public String generateSparkJarsPaths(List<String> commandLineJarsPaths) {
-    @Override
-    public boolean doSupportNewHBaseAPI() {
-        return true;
-    }
```
Sorry, really be mistake. Will correct that.

sponomarova commented:
> pleaase do not remove the jenkinsfile

Will revert this. Thanks

sponomarova commented:
> this commit needs to be reverted please : [b20162b](https://github.com/Talend/tbd-studio-se/commit/b20162b3f08e2ae37dfa64dc1d74db912b4221e4)

Partially reverted.

sponomarova commented on README.md:
```diff
@@ -1,57 +0,0 @@
-# Talend Open Studio for Big Data
-http://www.talend.com
-
-
```
Reverted.

sponomarova commented on main/plugins/org.talend.hadoop.distribution.apache100/src/main/java/org/talend/hadoop/distribution/apache100/Apache100Distribution.java:
```diff
@@ -117,22 +107,7 @@ public boolean doSupportCrossPlatformSubmission() {
-    @Override
-    public boolean doSupportHCatalog() {
-        return false;
-    }
```
`doSupportHCatalog()` is used for PigComponent interface to be able to interact with HCatalog wia pig scripts. 
Since pig is about to be removed this API is not in use.

sponomarova commented:
> * main/plugins/org.talend.designer.components.bigdata/components/tPigLoad/*
> * main/plugins/org.talend.designer.components.bigdata/components/tPigStoreResults/*
> 
> should be removed as well

Will be removed.

aparent-talend commented on main/plugins/org.talend.designer.components.bigdata/components/tCassandraBulkExec/tCassandraBulkExec_java.xml:
```diff
@@ -246,8 +246,7 @@
             <IMPORT
                 NAME="snappy-java-1.0.5"
-                MODULE="snappy-java-1.0.5.jar" MVN="mvn:org.talend.libraries/snappy-java-1.0.5/6.0.0" 
-                UrlPath="platform:/plugin/org.talend.libraries.pig/lib/snappy-java-1.0.5.jar"
```
Why this change ? It's not related to pig ?

sponomarova replied:
@aparent-talend , this change is related to pig. 
As you may see, UrlPath points on "org.talend.libraries.pig" which is removed in scope of this feature/work item.
Also I removed "UrlPath" parameter since it is optional.

The same reasons may be applied to other components where similar changes were done.

aparent-talend commented on main/plugins/org.talend.hadoop.distribution.emr550/src/main/java/org/talend/hadoop/distribution/emr550/EMR550Distribution.java:
```diff
@@ -485,4 +452,10 @@ public boolean doSupportAvroDeflateProperties(){
     }
+
+	@Override
+	public boolean doSupportNewHBaseAPI() {
```
why this addition ?

sponomarova replied:
Somehow this code was removed under earlier merge with master.
However, this should be added since this code is in master:
https://github.com/Talend/tbd-studio-se/blob/e6995a0a37adb7f2a3b821f482d4688b9b89a986/main/plugins/org.talend.hadoop.distribution.emr550/src/main/java/org/talend/hadoop/distribution/emr550/EMR550Distribution.java#L303
